workers/emailer/lib/emailer/emailerworker.coffee

```diff
@@ -74,22 +74,20 @@ module.exports = class EmailerWorker
 
   sendMail: (message) ->
 
+    to = message.Properties.Username
     type = getEmailType message.Subject
+    template = (TEMPLATES message.Properties.Options)[type]
 
-    message.Properties.Options.username = message.Properties.Username
```

Comment:
this line is required for templates ref: https://github.com/koding/koding/commit/ac61b996cd6bb4ff7dc6d517626453af4f679cd2